packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -238,6 +238,22 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
     super.didChangeDependencies();
   }
 
+  @override
+  void didUpdateWidget(covariant RefreshIndicator oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    if (oldWidget.color != widget.color) {
+      final ThemeData theme = Theme.of(context);
+      _valueColor = _positionController.drive(
+        ColorTween(
+          begin: (widget.color ?? theme.accentColor).withOpacity(0.0),
+          end: (widget.color ?? theme.accentColor).withOpacity(1.0),
+        ).chain(CurveTween(
+            curve: const Interval(0.0, 1.0 / _kDragSizeFactorLimit)
+        )),
+      );
+    }
```

Comment:
We might think about refactoring this out from here and `didChanceDependencies` to reduce some duplication, but it it is not critical.

Reply:
I really thought about that, will send a fix today.

Reply:
Given that this is already in, I wouldn't worry about it for this one. If we touch the file again we can take another look.